client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -139,6 +139,7 @@ private void initializeLifecycleManager(String appId) {
         if (lifecycleManager == null) {
           appUniqueId = celebornConf.appUniqueIdWithUUIDSuffix(appId);
           lifecycleManager = new LifecycleManager(appUniqueId, celebornConf);
+          lifecycleManager.applicationCount().increment();
```

Comment:
rebased the code, could you take a look? @RexXiong

Reply:
This comments should has been resolved.
https://github.com/apache/celeborn/pull/3026#discussion_r1912595366